components/tidb_query_datatype/src/expr/ctx.rs

```diff
@@ -108,6 +114,7 @@ impl EvalConfig {
             max_warning_cnt: DEFAULT_MAX_WARNING_CNT,
             sql_mode: SqlMode::empty(),
             paging_size: None,
+            div_precision_increment: DEFAULT_DIV_PRECISION_INCR,
```

Comment:
Instead of adding a new default value, how about using the existing value `DEFAULT_DIV_FRAC_INCR`?

Reply:
Nice, changed it.